src/enums/EnforceMfa.php

```diff
@@ -0,0 +1,43 @@
+<?php declare(strict_types=1);
+/**
+ * @author Nicolas CARPi <nico-git@deltablot.email>
+ * @author Marcel Bolten <github@marcelbolten.de>
+ * @copyright 2023 Nicolas CARPi
+ * @see https://www.elabftw.net Official website
+ * @license AGPL-3.0
+ * @package elabftw
+ */
+
+namespace Elabftw\Enums;
+
+use function array_column;
+use function array_combine;
+use function array_map;
+
+enum EnforceMfa: int
+{
+    case Disabled = 0;
+    case Admins = 1;
+    case Users = 2;
+    case Everyone = 3;
```

Comment:
I would go with "Sysadmins", "Admins", "Everyone". It doesn't make sense to enforce for users but not admins!